src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/blogs/BlogsPage.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:wicket="http://wicket.apache.org/dtds.data/wicket-xhtml1.5-strict.dtd">
+
+<head></head>
+
+<body>
+
+<wicket:extend>
+
+    <div id="page-wrapper">
+        <div class="row">
+            <div class="col-lg-12">
+                <h1 class="page-header">Blogs Managment Panel</h1>
+            </div>
+            <!-- /.col-lg-12 -->
+        </div>
+
+        <!-- /.row -->
+        <div class="row">
+            <div class="col-lg-12">
+                <div class="panel panel-primary">
+                    <div class="panel-heading">
+                        <i class="fa fa-rss fa-fw"></i> Latest blogs
+                    </div>
+                    <!-- /.panel-heading -->
+                    <div class="panel-body">
+                        <div wicket:id="feedback"></div>
+                        <form wicket:id="blogDataForm">
+                            <table cellspacing="0" class="table table-striped table-bordered table-hover dataTable no-footer">
+                                <tr>
+                                    <th>Author</th>
+                                    <th>RSS</th>
+                                    <th>Twitter</th>
+                                    <th>Date added</th>
+                                    <th>Status</th>
+                                    <th>Actions</th>
+                                </tr>
+                                <tr wicket:id="blogsDataView" class="gradeA">
+                                    <td><span wicket:id="author">[author]</span></td>
+                                    <td><a href="" wicket:id="rss" target="_blank">[rss]</a></td>
+                                    <td><a href="" wicket:id="twitterLink" target="_blank">[twitterLink]</a></td>
+                                    <td><span wicket:id="dateAdded">[dateAdded]</span></td>
```

Comment:
Contents of dateAdded, status and actions columns should be centered (like in Moderation Page)